pkg/rest/request.go

```diff
@@ -71,14 +73,26 @@ func requestWithBody(method string, url string, body io.Reader, apiToken string)
 
 func executeRequest(request *http.Request) Response {
 	client := &http.Client{}
-	resp, err := client.Do(request)
-	if err != nil {
-		println("HTTP Request failed with Error: " + err.Error())
-		return Response{}
-	}
-	defer func() {
-		err = resp.Body.Close()
-	}()
-	body, err := ioutil.ReadAll(resp.Body)
-	return Response{resp.StatusCode, body}
+
+	rateLimitStrategy := createRateLimitStrategy()
+
+	// TODO properly handle error
+	response, _ := rateLimitStrategy.executeRequest(util.NewTimelineProvider(), func() Response {
+		resp, err := client.Do(request)
+		if err != nil {
+			println("HTTP Request failed with Error: " + err.Error())
```

Comment:
just noticed this uses a println and not the Logger, could you please change that while you're changing things? :)

Reply:
Sure thing, good catch. Eagle-eye @UnseenWizzard.

Reply:
Done in 3fa1d52.